src/test/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheckTest.java

```diff
@@ -86,7 +86,7 @@ public void testIncorrectAllTokens() throws Exception {
         checkConfig.addAttribute("tokens", "CLASS_DEF, INTERFACE_DEF, ENUM_DEF, METHOD_DEF, "
                 + "CTOR_DEF, VARIABLE_DEF, PARAMETER_DEF, ANNOTATION_DEF, TYPECAST, "
                 + "LITERAL_THROWS, IMPLEMENTS_CLAUSE, TYPE_ARGUMENT, LITERAL_NEW, DOT, "
-                + "ANNOTATION_FIELD_DEF");
+                + "ANNOTATION_FIELD_DEF, ENUM_CONSTANT_DEF");
```

Comment:
test is named `testIncorrectAllTokens ` probably we need to add all tokens.
and we probably need to update `InputAnnotationLocationIncorrect.java` with some cases.

Reply:
Yes I added this test for pitest. All tokens should be added.

Reply:
done.
Actually, `PACKAGE_DEF` is irrelevant here. All package annotations should be in the file `package-info.java`.